Citrix/citrix-adc/tests/test_blocked_event.json

```diff
@@ -0,0 +1,54 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Citrix ADC",
+        "dialect_uuid": "02a74ceb-a9b0-467c-97d1-588319e39d71"
+      }
+    },
+    "message": "CEF:0|Citrix|NetScaler|NS13.1|APPFW|APPFW_MULTIPLE_HEADER|6|src=1.2.3.4 geolocation=Unknown spt=61903 method=GET request=https://www.example.org/services msg=Multiple headers in request cn1=3755128 cn2=1636674 cs1=WAF_PRF_RULE1 cs2=PPE1 cs4=ALERT cs5=2023 act=blocked"
+  },
+  "expected": {
+    "message": "CEF:0|Citrix|NetScaler|NS13.1|APPFW|APPFW_MULTIPLE_HEADER|6|src=1.2.3.4 geolocation=Unknown spt=61903 method=GET request=https://www.example.org/services msg=Multiple headers in request cn1=3755128 cn2=1636674 cs1=WAF_PRF_RULE1 cs2=PPE1 cs4=ALERT cs5=2023 act=blocked",
```

Comment:
Could you extract the field `request` into `http.request.method`?

Reply:
@squioc you mean `method` in `http.request.method` because `request` is already in `url.original` ?

Reply:
@TOUFIKIzakarya My bad. Yes, that was method.